### PR TITLE
Footer updates: Add a link to the repo.

### LINKED ISSUE #924: Add GitHub icon in footer?
Currently, I don't think there is an easy way [in the v2 design] to find the GitHub repo if you want to contribute or submit a bug.

### V1 footer
Firefox_Monitor

### V2 footer
Firefox_Monitor

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
Closes #924 

Hopefully this will make it easier for people to find the repo. 
Maybe we also want to consider #125?

### CODE REVIEW

File: locales/en/app.ftl [hunk 1]
```diff
@@ -12,6 +12,7 @@
 -brand-HIBP = Have I Been Pwned
 -brand-fxa = Firefox Account
 -brand-Chrome = Chrome
+-github = Github
```
Comment (flodolo) [inline locales/en/app.ftl]: That's not how you write GitHub ;-)

File: locales/en/app.ftl [hunk 2]
```diff
@@ -23,6 +24,8 @@ about-firefox-alerts = About Firefox Alerts
 give-feedback = Give Feedback
 terms-and-privacy = Terms & Privacy
 
+github-link-title = {-github}
```
Comment (flodolo) [inline locales/en/app.ftl]: If you're only using it once, you shouldn't create a term for it, just 


```
github-link-title = GitHub
```

File: locales/en/app.ftl (hunk 1 above)
Comment (lesleyjanenorton) [reply locales/en/app.ftl]: thank you!

**Comment by groovecoder** [comment]
We shouldn't need to tag it with utm params ourselves. [Our GitHub repo traffic page](https://github.com/mozilla/blurts-server/graphs/traffic) already shows referring sites.

Or do you mean for us to see how many visitors actually click this?

**Pull request merged**